src/controller/login/postLogin.js

```diff
@@ -1 +1,44 @@
+const hashPassword = require("../../utils/hash");
+const findUser = require("../../database/queries/selectUser");
+const { create } = require("../../utils/cookie");
 
+module.exports = async (req, res) => {
+  try {
+    const {
+      data: { email, password }
+    } = req.body;
+
+    // Check if user exists
+    const user = await findUser({ email });
+    if (!user) throw Error("Email or password were incorrect");
+    console.log({ user });
+
+    const hashedPassword = await hashPassword(password);
```

Comment:
You should not hash the password again and compare it to the hashed password stored in the database using "===" since they will never match. `bcrypt` generates random and different salf each time you use `hash` method. You should use `compare` method. 
In utils:
```suggestion
    module.exports = (pass1, pass2) => bcrypt.compare(pass1, pass2);
```
pass2 is the hashed password in the database and pass1 is the password entered by the user in login form.

Reply:
ahh I totally forgot